sw/otbn/code-snippets/p256.s

```diff
@@ -1405,70 +1229,54 @@ p256_sign:
  * This routine runs in constant time.
  *
  * @param[in]  dmem[4]: dptr_rnd, pointer to location in dmem containing random
- *                        number for blinding
+ *                      number for blinding
```

Comment:
The indentation used a couple lines below would be to indent "number for blinding" under "pointer". Fine either way.